i18n/en.json

```diff
@@ -1448,6 +1448,18 @@
     "id": "api.post.create_post.town_square_read_only",
     "translation": "This channel is read-only. Only members with permission can post here."
   },
+  {
+    "id": "api.post.patch_post.can_not_udpate_post_in_deleted.error",
+    "translation": "Can not update a post in a deleted channel."
+  },
+  {
+    "id": "api.post.udpate_post.can_not_udpate_post_in_deleted.error",
```

Comment:
Typo in the translation key name `udpate` (in 2 places). Also, could `api.post.udpate_post.can_not_udpate_post_in_deleted.error` and `api.post.patch_post.can_not_udpate_post_in_deleted.error` simply share the same translation?

Reply:
Yeah, I going to fix it right away

Reply:
Fixed!